src/lib/layout/SubHeader.tsx

```diff
@@ -6,106 +6,150 @@ import {
   useMoveConfig,
   useNftConfig,
   usePoolConfig,
+  useTierConfig,
   useWasmConfig,
 } from "lib/app-provider";
 import { AppLink } from "lib/components/AppLink";
 import type { IconKeys } from "lib/components/icon";
 import { CustomIcon } from "lib/components/icon";
 import { useIsCurrentPage } from "lib/hooks";
 
+const ACTIVE_COLOR = "primary.light";
+
 interface SubHeaderMenuInfo {
   name: string;
   slug: string;
   icon: IconKeys;
 }
 
 const SubHeader = () => {
+  const tier = useTierConfig();
   const govConfig = useGovConfig({ shouldRedirect: false });
   const wasmConfig = useWasmConfig({ shouldRedirect: false });
   const moveConfig = useMoveConfig({ shouldRedirect: false });
   const nftConfig = useNftConfig({ shouldRedirect: false });
   const poolConfig = usePoolConfig({ shouldRedirect: false });
 
-  const subHeaderMenu: SubHeaderMenuInfo[] = [
-    { name: "Overview", slug: "/", icon: "home" },
-    { name: "Transactions", slug: "/txs", icon: "file" },
-    { name: "Blocks", slug: "/blocks", icon: "block" },
-  ];
-
-  if (govConfig.enabled)
-    subHeaderMenu.push(
-      { name: "Validators", slug: "/validators", icon: "validator" },
-      { name: "Proposals", slug: "/proposals", icon: "proposal" }
-    );
-
-  if (wasmConfig.enabled)
-    subHeaderMenu.push(
-      { name: "Codes", slug: "/codes", icon: "code" },
-      { name: "Contracts", slug: "/contracts", icon: "contract-address" }
-    );
-
-  if (moveConfig.enabled)
-    subHeaderMenu.push({
-      name: "Modules",
-      slug: "/modules",
-      icon: "contract-address",
-    });
-
-  if (nftConfig.enabled)
-    subHeaderMenu.push({
-      name: "NFTs",
-      slug: "/nft-collections",
-      icon: "group",
-    });
-
-  if (poolConfig.enabled)
-    subHeaderMenu.push({ name: "Osmosis Pools", slug: "/pools", icon: "pool" });
-
   const isCurrentPage = useIsCurrentPage();
 
-  const activeColor = "primary.light";
-
-  const trackOnClick = (tab: string) => {
-    track(AmpEvent.USE_TOPBAR, { tab });
-  };
+  const subMenu: SubHeaderMenuInfo[] =
```

Comment:
Shall we make an objects for tier `full` and `lite` navigation bar separately. So it will be more easy to read?